tests/x11regressions/firefox/firefox_java.pm

```diff
@@ -20,6 +20,7 @@ sub java_testing {
     my ($self) = @_;
 
     send_key "ctrl-t";
+    wait_still_screen 3;
```

Comment:
Can we apply any reasonable screen assertion here? Because, as practice shows, wait_screen_change and wait_still_screen are not helping in many scenarios, for example when change is small. Or 3 seconds is actually not enough. Otherwise, let's merge it and see if it helps.

Reply:
Agreeing with @rwx788 here. I think you are using `wait_still_screen` for something it was not intended for, it should not be a "lazy screen check" in case we want to avoid a proper needle check. A proper `assert_screen` might be a bit higher needle maintenance effort but we save at least 3 seconds on every call and also have a more robust test.

Reply:
Wait was replaced with new needle match for Firefox java new tab and Firefox private browsing.
https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/577

Private browsing: http://10.100.12.105/tests/591#step/firefox_private/9
New tab: http://10.100.12.105/tests/591#step/firefox_java/14